<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="credit">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;a href=&quot;https://github.com/TheKoopaKingdom/CHMOD-Generator&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Made with love by TheKoopaKingdom&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="inputLay">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="browseLay">
       <item>
        <widget class="QPushButton" name="browse">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control + B&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="pathOut">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2">
      <layout class="QVBoxLayout" name="optionsLay">
       <item>
        <widget class="QCheckBox" name="sub">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control + S&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Subdirectorys/files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="force">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control + F&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Force</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="verbose">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control + V&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Verbose (feedback window)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTabWidget" name="permissionsTab">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="comBoxTab">
        <attribute name="title">
         <string>Combo Box</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="rootText">
           <property name="text">
            <string>Root Access:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="rootBox"/>
         </item>
         <item>
          <widget class="QLabel" name="userText">
           <property name="text">
            <string>User Access:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="userBox"/>
         </item>
         <item>
          <widget class="QLabel" name="worldText">
           <property name="text">
            <string>World Access:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="worldBox"/>
         </item>
        </layout>
        <zorder>rootBox</zorder>
        <zorder>userText</zorder>
        <zorder>userBox</zorder>
        <zorder>worldBox</zorder>
        <zorder>worldText</zorder>
        <zorder>rootText</zorder>
       </widget>
       <widget class="QWidget" name="cheBoxTab">
        <attribute name="title">
         <string>Check Boxes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTabWidget" name="cheBoxTabs">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="rootTab">
            <attribute name="title">
             <string>Root</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="rReadBox">
               <property name="text">
                <string>Read</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="rWriteBox">
               <property name="text">
                <string>Write</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="rExeBox">
               <property name="text">
                <string>Execute</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="userTab">
            <attribute name="title">
             <string>User</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCheckBox" name="uReadBox">
               <property name="text">
                <string>Read</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="uExeBox">
               <property name="text">
                <string>Execute</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="uWriteBox">
               <property name="text">
                <string>Write</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="worldTab">
            <attribute name="title">
             <string>World</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="wReadBox">
               <property name="text">
                <string>Read</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="wExeBox">
               <property name="text">
                <string>Execute</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="wWriteBox">
               <property name="text">
                <string>Write</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="outputLay">
     <item>
      <widget class="QPushButton" name="gen">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control + G&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="output">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="copyRun">
       <item>
        <widget class="QPushButton" name="run">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control + R&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copy">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control + C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>browse</tabstop>
  <tabstop>gen</tabstop>
  <tabstop>run</tabstop>
  <tabstop>copy</tabstop>
  <tabstop>pathOut</tabstop>
  <tabstop>userBox</tabstop>
  <tabstop>worldBox</tabstop>
  <tabstop>sub</tabstop>
  <tabstop>force</tabstop>
  <tabstop>verbose</tabstop>
  <tabstop>output</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
